.github/workflows/create-jira-issues-for-community-activities.yml

```diff
@@ -2,15 +2,16 @@ name: Create Jira issues for community activities
 
 on:
   discussion:
-    types: created
+    types: [created]
   issues:
-    types: opened
+    types: [opened]
   pull_request:
-    types: opened
+    types: [opened]
 
 jobs:
   call-publish-workflow:
-    uses: Lombiq/GitHub-Actions/.github/workflows/create-jira-issues-for-community-activities.yml@dev
+    uses: Lombiq/GitHub-Actions/.github/workflows/create-jira-issues-for-community-activities.yml@issue/OSOE-485
+    name: Call publish workflow
```

Comment:
These additional names are not required, here and in the other files.

Reply:
Still, please delete it.

Reply:
deleted